pkg/capi/edit/cluster.x-k8s.io.cluster/ClusterConfig.vue

```diff
@@ -550,210 +574,212 @@ export default {
       </CardGrid>
     </template>
     <template #stepConfiguration>
-      <Accordion
-        class="mt-20 section-accordion"
-        open-initially
-        :title="t(`capi.cluster.section.${formSections.GENERAL}`)"
-      >
-        <!-- GENERAL CONFIGURATION -->
-        <NameNsDescription
-          v-if="!isView"
-          :value="value"
-          :mode="mode"
-          :namespaced="classNamespaceSupported"
-          :namespace-options="allNamespaces"
-          name-label="cluster.name.label"
-          name-placeholder="cluster.name.placeholder"
-          description-label="cluster.description.label"
-          description-placeholder="cluster.description.placeholder"
-          :rules="{ name: fvGetAndReportPathRules('metadata.name') }"
-          @update:value="$emit('update:value', { k: 'metadata', val: $event.metadata })"
-        />
-        <div class="mt-30">
-          <Checkbox
-            v-model:value="autoImport"
+      <div :key="topology.class">
```

Comment:
This diff looks crazy but I just added a div here. The key on this div should force all its children to re-initialize when `class` changes.